src/components/views/rooms/RoomHeader.tsx

```
@@ -159,14 +159,14 @@ interface VideoCallButtonProps {
     behavior: DisabledWithReason | "legacy_or_jitsi" | "element" | "jitsi_or_element";
 }
 
-function QRCodeModal({ show, onClose, qrData, checkbox,room,destination,amount,currency }) {
+function QRCodeModal({ show, onClose, qrData, checkbox,room,destination,amount,currency,sender,explorer }) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)